src/App.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react'
+import TimecodeTracker from './components/TimecodeTracker/TimecodeTracker'
+import styled from 'styled-components'
 
-function App() {
-  return <h1>Hello World!</h1>
+export default function App() {
+  return (
+    <div>
+      <AppTitle>Timecode Tracker</AppTitle>
+      <TimecodeTracker />
+    </div>
+  )
 }
 
-export default App
+const AppTitle = styled.h2`
```

Comment:
Why h2? If its your main app title it should be h1.

Reply:
changed it to h1. thx